learn/advanced-introduction/qi-privacy.mdx

```diff
@@ -0,0 +1,148 @@
+
+In short, **privacy is a shield for freedom, competition, and innovation.**  
+A truly global, decentralized economy must protect privacy *by design*, not as an afterthought.
+
+<Callout>
+💡 If we are to build a new global monetary system, privacy must be baked in by default — not left as an afterthought.
+</Callout>
+
+---
+
+## Cash-Like Privacy
+
+At Quai Network, we describe **QI** as having *cash-like privacy* because it mirrors the privacy dynamics of physical cash.
+
+> **Example:** When you pay with cash, both parties know the transaction occurred and agree on the amount.  
+> However:
+> - The merchant doesn’t know how much money you still have.  
+> - You don’t know how much cash the merchant has.  
+> - Observers can confirm a transaction occurred but cannot trace where the cash came from or where it ends up.
+
+This is **transparency without traceability** — a property QI achieves through its unique ledger design.
+
+---
+
+## The QI Ledger Privacy Model
+
+**QI** operates on a **[UTXO ledger](https://en.bitcoin.it/wiki/Transaction#Input_and_Output)**, similar to Bitcoin’s, but with two key differences:
+
+1. **Enforced Non-Address Reuse**
+2. **Payment Codes**
+2. **Fixed Denominations**
+
+These modifications to the traditional UTXO model make QI transactions private *by default* while maintaining global scalability.
+
+---
+
+### 1. Enforced Non-Address Reuse
+
+On the QI ledger, **addresses cannot be reused.**
+
+Every transaction to a recipient must generate a **unique address**. This prevents observers from tracking user activity by linking multiple payments to the same address — a common weakness in blockchain systems like Bitcoin where address reuse compromises privacy.
+
+<Callout type="warning">
+⚠️ One address may be reused once per prime block for UTXO consolidation, improving efficiency while maintaining privacy.
+</Callout>
+
+---
+
+#### 2. Payment Codes: Reusable Identities Without Address Reuse
+
+**[Payment Codes](https://github.com/bitcoin/bips/blob/master/bip-0047.mediawiki)** are a cryptographic mechanism that allows senders to derive unique addresses for each transaction while the recipient maintains a single, reusable identifier.
+
+This enables **reusable identities with unique transaction addresses** — giving QI users the convenience of a persistent payment identifier while maintaining cash-like privacy by default.
+
+---
+
+### 3. Fixed Denominations
+
+While non-reuse hides *who* is transacting, **fixed denominations** hide *how much* and *where it goes*.
+
+Unlike Bitcoin, which supports 100 million *satoshis* per BTC (creating traceable patterns), **QI** limits the system to just **16 fixed denominations**.
+
+This design means most transactions will require inputs from both parties making the default behavior look like a 2 person coinjoin. 
+
+Together, **enforced non-address reuse**, **payment codes**, and **fixed denominations** make QI transactions **effectively untraceable**.
```

Comment:
Also, mention here, that if people want to increase the rate at which entropy is generated in the set they can optionally use secondary tools to do many party coinjoins.